### ISSUE TITLE
 Why no mac-intel-x86 version tensorflow after tensorflow 2.16.2?

### ISSUE THREAD

**Issue by sexjun** [tensorflow/tensorflow#78836]
### Issue type

Build/Install

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

binary

### TensorFlow version

2.18.0

### Custom code

No

### OS platform and distribution

mac 15.0.1

### Python version

3.10

### Current behavior?

I can't install tensorflow 2.17 2.18 version because there is no relase version for my sytem.
does no support mac intel chip anymore after 2.16.2 version?
<img width="477" alt="image" src="https://github.com/user-attachments/assets/dbe20bf6-c6cd-4f0e-982d-3b80f42543c9">

### Standalone code to reproduce the issue

```shell
no code
I can pip_search tensroflow 2.18
but only 2.16.2 can been downloaded.
and no wheel file can found .
```

**Comment by sexjun** [comment]
mac intel chip anaconda found 2.17 version tensroflow. 
it sloved my problem. iree work expect.

does google will not support mac with intel chip after tensorflow 2.17 but community does?
```
(tf) ➜  offcial_guide_tf git:(20241017) ✗ conda  list | grep tensorflow
tensorflow                2.17.0          cpu_py310hb3d9b59_0
tensorflow-base           2.17.0          cpu_py310ha37eea7_0
tensorflow-estimator      2.17.0          cpu_py310h7aa49ff_0
```

**Comment by tilakrayal** [comment]
@sexjun,
Have you got the chance to have a look at the official release document where it was mentioned that Mac x86 builds are being deprecated and will no longer be released as a Pip package from TF 2.17 onwards.
https://github.com/tensorflow/tensorflow/releases/tag/v2.16.1

Thank you!

**Comment by sexjun** [comment]
@tilakrayal 
Thanks very much , That's what i need.
